src/routes/Home/Home.test.js

```diff
@@ -0,0 +1,11 @@
+import { render as rtlRender, screen } from '@testing-library/react';
```

Comment:
Am in a hotel in Chicago (made it though all the border checks and covid paperwork, yay!)

The tests themselves look good here, but this import is a little unfamiliar. If yr importing render aliased as rtlRender then you might need to call it as rtlRender on line 7?

If that's not it, what's the exact error you get?